src/static/_locales/en/messages.json

```diff
@@ -391,6 +391,10 @@
 		"message": "Cancel",
 		"description": "'Cancel' button in image picker"
 	},
+	"pickerGroupCounterClicked": {
```

Comment:
In this project, message ID is constructed with scope / item / type. It would be better to use an ID like `pickerTabCounterTitle`. If we switch to a button, it should be `pickerTabInvert` and `pickerTabInvertTitle`. (Though I think there is no need to use a title if we remove the shift key logic)